### ISSUE TITLE
Paper dialog does not display properly when the dialog is opened immedietly when created

### ISSUE THREAD

**Issue by KrauseStefan** [PolymerElements/paper-dialog#80]
The problem occurs when the dialog is immediately opened during creation either by script or by the opened attribute.

I've created a fiddle based on the demo, that illustrates the problem.
http://jsfiddle.net/krausekjaer/z7qsjr09/

Try removing the opened attribute and click the button and it all works like it is supposed to.

Currently I'm using this workaround to make it work in my angular2 app:

``` javascript
  ngAfterViewInit() {
    window.setTimeout(() => {
      if(!this.getDialogElement().opened){
        this.getDialogElement().open();
      }
      window.setTimeout(() => {
        this.getDialogElement().fit();
      });
    });
  }
```

It might be that it is not intended to be used in this way, but would be nice it the opened attribute worked for this scenario. Please advice :)

**Comment by valdrinkoshi** [comment]
you should import also webcomponents

``` html
<script src="http://polygit.org/components/webcomponentsjs/webcomponents-lite.min.js"></script>
```

http://jsbin.com/korapu/edit?html,output

If you use `paper-dialog-scrollable` and set the `paper-dialog` opened on creation, you won't see its content (bug already opened here https://github.com/PolymerElements/paper-dialog-scrollable/issues/13)
Closing this

**Comment by KrauseStefan** [comment]
webcomponets-lide.js is imported as a external resource.

The reason why I filled the issue here was that I get a similar error without <paper-dialog-scrollable>.
I included the <paper-dialog-scrollable> in order to show the most extreme version. But it might be two different issues.

http://jsfiddle.net/Luotbhch/2/

The dialog is not correctly positioned and sized when using the opened attribute.

Edit:
Sorry I was a little too quick, I see that your example actually works, however I have a hard time replicating your results. Will comment if I don't succeed later.

**Comment by KrauseStefan** [comment]
The error was the version of web components, I had referenced the tip of the master branch instead of the released version (0.7.20). I still see the error on my local version but cannot reproduce outside my application.

https://raw.githubusercontent.com/webcomponents/webcomponentsjs/ce9c8597696ec4bafc772c2126a3b1c7b0e948c0/webcomponents-lite.js

vs 
https://raw.githubusercontent.com/webcomponents/webcomponentsjs/master/webcomponents-lite.js

Thank you for your help

**Comment by valdrinkoshi** [comment]
It seems more of an issue with your current setup, or something cached between your dependencies. 
Have you tried to delete the bower folder and run again `bower install` ? (I assume you're using bower)

**Comment by KrauseStefan** [comment]
Agreed I just wanted to put up the info that I referenced the wrong file in my jsfiddle.
Yes I have done so, but I'm sure that it is my own fault.

Currently it is a chrome only issue, so it might be related to the the native implementation of "shadow dom or simular"

I use the following to snippet to make it work with Angular 2

``` javascript
//https://github.com/Polymer/polymer/issues/2526
window.Polymer = window.Polymer || {};
window.Polymer.dom = 'shadow';
```

I've been trying to make a new fiddle where I integrated it with angular2 but I don't get the error, however I see it just like when I use webcomponents-lite.js from https://raw.githubusercontent.com/webcomponents/webcomponentsjs/master/webcomponents-lite.js

I've been trying to creating a jsfiddle with Angular 2 but I can't get it to work properly with a configuration similar to my local one (not in a few hours anyway). 
I'm going to keep using my work around for now, until I find a proper way to reproduce.
